server/sonar-server/src/test/java/org/sonar/server/component/index/ComponentIndexScoreTest.java

```diff
@@ -76,13 +77,15 @@ public void scoring_prefix_with_multiple_words_and_case() {
       "index_java.js");
   }
 
+  @Ignore
   @Test
   public void scoring_long_items() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Fix or remove this skipped unit test [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1607)